### PR TITLE
Code refactoring

### PR DESCRIPTION
**Pull request by anasm11**
added video page (VideoPage.jsx) and refactored code

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,18 +1,31 @@
 @import url("https://lucidui.netlify.app/library.css");
 
-
-
 .nav{
     position:sticky;
     top:0;left:0;right:0;
+    z-index: 999;
```
Comment (nikhiljugale007) [inline src/App.css]: try not to use this much higher z-index. 10-20 should work.

File: src/App.css [hunk 2]
```diff
@@ -1,18 +1,31 @@
...
+}
+
+.card{
+    width:300px;
```
Comment (nikhiljugale007) [inline src/App.css]: try to use rems instead of px.

File: src/backend/db/videos.js [hunk 3]
```diff
@@ -4,28 +4,85 @@
...
+  //   description:
+  //     "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
+  //   creator: "Soham Shah"
+  // },
+  // {
+  //   _id: "F_Riqjdh2oM",
+  //   title: "Neural Networks from Scratch - P.1 Intro and Neuron Code",
+  //   creator: "Sentdex",
+  //   description:
+  //     "At ero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga."
```
Comment (nikhiljugale007) [inline src/backend/db/videos.js]: remove commented code before pushing of not required.

File: src/components/PlaylistModal.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                    <div className="text-container">
+                        <div className="title-text">Add to Playlist</div>
+                        <div className="author-text"></div>
+                    </div>
+
+
+                    <div className="description"><input onChange={(e) => setNewPlaylistName(e.target.value)} /><button onClick={async () => {
+                        (async () => {
+                            const res = await axios({
+                                method: 'post',
```
Comment (nikhiljugale007) [inline src/components/PlaylistModal.js]: suggestion: try to create a separate file for API calls.

**Review by nikhiljugale007 — COMMENTED** [review]
Try to format your code before pushing.
Added a few suggestions.

File: src/pages/VideoPage.jsx [hunk 5]
```diff
@@ -1,7 +1,99 @@
...
+            </div>
+
+            <div className='secondary'>
+                {videos.map((video)=>
+                <div class="card horizontal">
+                <div class="card-body">
+                  <div class="img-text">
+                    <img src={video.img} />
+                    <div class="text-container">
+                      <div class="title-text">{video.title}</div>
```
Comment (yaswanthmyneni) [inline src/pages/VideoPage.jsx]: suggestion: create a separate component for this card

File: src/pages/WatchLater.jsx [hunk 6]
```diff
@@ -1,40 +1,37 @@
...
+  const { watchLaterVideos, watchLaterDispatch } = useWatchLaterContext()
+  return (
+    <div>
+      {watchLaterVideos.map((video) =>
+        <div className="card" key={video._id}>
+          <div className="card-body">
 
+            <div className="text-container">
+              <div className="title-text">{video.title}</div>
+              <div className="author-text">{video.creator}</div>
```
Comment (yaswanthmyneni) [inline src/pages/WatchLater.jsx]: suggestion: same here, create a component for this card

**Review by yaswanthmyneni — APPROVED** [review]
Given suggestions, otherwise, you can merge the pr.

**Pull request merged**